src/app/visit/visitStatus.cpp

```diff
@@ -0,0 +1,20 @@
+#include "visitStatus.hpp"
+
+std::string toString(VisitStatus status)
+{
+    switch (status)
+    {
+    case VisitStatus::Canceled:
+        return "Canceled";
+
+    case VisitStatus::Confirmed:
+        return "Confirmed";
+
+    case VisitStatus::Relocated:
+        return "Relocated";
+
+    case VisitStatus::Visited:
+        return "Visited";
+    }
+    return "Unassigned";
```

Comment:
"Unassigned" should be one of cases and valid enum values.
BTW need for any kind of "default" (like line 19) behaviors for checking enums  classes -> they can have only defined values.